dtfs-central-api/api-tests/mocks/utilisation-reports/utilisation-report-raw-csv-data.ts

```diff
@@ -1,15 +1,16 @@
+import { CURRENCY } from '@ukef/dtfs2-common';
 import { UtilisationReportRawCsvData } from '../../../src/types/utilisation-reports';
 
 export const MOCK_UTILISATION_REPORT_RAW_CSV_DATA: UtilisationReportRawCsvData = {
   'ukef facility id': '0123456789',
-  'base currency': 'GBP',
+  'base currency': CURRENCY.GBP,
```

Comment:
Thoughts on exporting `GBP` and other currencies?

I am not too fussed and probably `CURRENCY.GBP` is better, otherwise could seem like an ambiguous acronym especially when other currencies are in play

Reply:
In this case it's pretty short and it helps with explaining where it's coming from, but so is GBP.  I think leave it in this case?
And other currencies should be another PR.  This ticket is already huge and is for GBP

Reply:
Agree

Do we have another ticket for non-GBP currencies?

Reply:
not that I know of currently, this one came from E2ETF for some reason

Reply:
Can we create at least 1x placeholder ticket please - looks like there is just EUR, JPY, USD after this PR.

I could pick these up actually, if you assign to me 😄  I can split the ticket into 3 tickets/PRs

Reply:
@ttbarnes do you mind picking this up plaease in the interest of closing this PR please.

Reply:
I've created a new ticket, DTFS2-7631 and renamed this PR since the original ticket number (DTFS2-3607) was unrelated to these changes - 3607 is a "sprint test execution" ticket.

This PR is one of 4 sub tasks, I will progess the others once this PR is merged.